### PR TITLE
VP-7030: Refactor all inputs in all details blades to register dynamically and render in va-metaform

### PR DESCRIPTION
**Pull request by eMazeika**
Jira: https://virtocommerce.atlassian.net/browse/VP-7030

### CODE REVIEW

**Comment by Vectorfield4** [comment]
Merge after https://github.com/VirtoCommerce/vc-platform/pull/2163

File: src/VirtoCommerce.CustomerModule.Web/Scripts/blades/employee-detail.tpl.html [hunk 1]
```diff
@@ -1,114 +1,83 @@
...
-            </form>
-        </div>
-    </div>
-</div>
+<div class="blade-static __bottom" ng-include="'$(Platform)/Scripts/common/templates/create.tpl.html'" ng-if="blade.isNew"></div>
+<div class="blade-content">
+    <div class="blade-inner">
+        <div class="inner-block clearfix" ng-controller="virtoCommerce.customerModule.employeeDetailController">
+            <form class="form" name="detailForm">
+                <!-- hide "isActive" switch as there is no functionality under it yet-->
```
Comment (Vectorfield4) [inline src/VirtoCommerce.CustomerModule.Web/Scripts/blades/employee-detail.tpl.html:6]: Remove all commented code, please.

**Review by Vectorfield4 — DISMISSED** [review]

File: src/VirtoCommerce.CustomerModule.Web/Scripts/blades/employee-detail.tpl.html [hunk 2]
```diff
@@ -1,114 +1,83 @@
...
+<script type="text/ng-template" id="timeZone.html">
+    <div class="form-input">
+        <ui-select ng-model="blade.currentEntity.timeZone">
+            <ui-select-match placeholder="{{'customer.blades.employee-detail.placeholders.timezone' | translate }}">{{$select.selected.name}}</ui-select-match>
+            <ui-select-choices repeat="x.id as x in blade.timeZones | filter: { name: $select.search }">
+                <span ng-bind-html="x.name | highlight: $select.search"></span>
+            </ui-select-choices>
+        </ui-select>
+    </div>
+</script>
```
Comment (pushnitsa) [inline src/VirtoCommerce.CustomerModule.Web/Scripts/blades/employee-detail.tpl.html:83]: Add empty line please

**Comment by Vectorfield4** [comment]
Do not merge before VirtoCommerce/vc-platform#2163

File: src/VirtoCommerce.CustomerModule.Web/module.manifest [hunk 3]
```diff
@@ -30,4 +30,4 @@
     <group>commerce</group>
   </groups>
   <useFullTypeNameInSwagger>false</useFullTypeNameInSwagger>
-</module>
\ No newline at end of file
+</module>
```
Comment (pushnitsa) [inline src/VirtoCommerce.CustomerModule.Web/module.manifest:33]: Add empty line please

**Review by pushnitsa — APPROVED** [review]

**Comment by Vectorfield4** [comment]
@eMazeika Can you provide a screenshot of the blade?

**Review by AliveMen — DISMISSED** [review]

**Comment by eMazeika** [comment]
> @eMazeika Can you provide a screenshot of the blade?

The requirement from related task: "No UI/UX changes".
Let's review 1 of the 4 affected blades. You can see the metaform is marked in red:

image

**Comment by Vectorfield4** [comment]
Looking good.

**Comment by Vectorfield4** [comment]
@eMazeika resolve branch conflicts, please.

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**